test/run-tests.py

```diff
@@ -237,6 +248,52 @@ def KillProcess(timeout=True):
   return stdout, stderr, returncode, duration
 
 
+def AsFlagsList(value):
```

Comment:
I prefer `args` to `flags` in general.  Maybe "SplitArgs"?

Reply:
Me too, but I wanted to do the flags -> args rename in another CL